CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Changelog
 
-## v1.0.0
```

Comment:
I've fixed the CHANGELOG (most recent release first), that's why the diff looks off.